src/getCoinCombination.test.js

```diff
@@ -1,10 +1,49 @@
-'use strict';
+"use strict";
 
-describe('getCoinCombination', () => {
-  const { getCoinCombination } = require('./getCoinCombination');
+describe(`getCoinCombination`, () => {
+  const { getCoinCombination } = require("./getCoinCombination");
 
   it(`should be declared`, () => {
-    expect(getCoinCombination)
-      .toBeInstanceOf(Function);
+    expect(getCoinCombination).toBeInstanceOf(Function);
+  });
+
+  it(`should be positive number`, () => {
+    expect(getCoinCombination(0)).toEqual([0, 0, 0, 0]);
+    expect(getCoinCombination(6)).toEqual([1, 1, 0, 0]);
+    expect(getCoinCombination(17)).toEqual([2, 1, 1, 0]);
+  });
+
+  it(`should be limit values`, () => {
+    expect(getCoinCombination(50)).toEqual([0, 0, 0, 2]);
+    expect(getCoinCombination(26)).toEqual([1, 0, 0, 1]);
+    expect(getCoinCombination(33)).toEqual([3, 1, 0, 1]);
+  });
```

Comment:
The description 'should be limit values' could be more descriptive. Consider renaming it to 'should handle edge cases correctly' to better reflect the purpose of these tests.